ospd_openvas/daemon.py

```diff
@@ -259,6 +259,9 @@ def __init__(self, *, niceness=None, **kwargs):
 
         self._niceness = str(niceness)
 
+        self.feed_lock_file = (
+            Path(kwargs.get('lock_file_dir')) / 'feed-update.lock'
```

Comment:
Personally I wouldn't use the kwargs dict. Instead i would add `lock_file_dir='some/default/file/path/'` as an argument to `__init__`